examples/aws/tidb-cluster.yaml

```diff
@@ -12,6 +12,14 @@ spec:
   pd:
     baseImage: pingcap/pd
     replicas: 3
+    service:
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+        service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+      exposeStatus: true
+      externalTrafficPolicy: Local
+      type: LoadBalancer
```

Comment:
Please do not expose the PD service as LB since PD does not support authentication/authorization yet, it's dangerous to do this.

Reply:
I'm afraid many users are not sufficiently aware of this risk as well. This might need a follow-up in the docs.

Instead of not exposing this (and maybe having users doing this themselves) can we expose this securely with an Ingress controller?

I haven't tested this, but something along the lines of this:
```yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  namespace: default
  name: ingress
  annotations:
    kubernetes.io/ingress.class: alb
    alb.ingress.kubernetes.io/scheme: internet-facing
spec:
  rules:
    - http:
        - path: /dashboard
          backend:
            serviceName: pd
            servicePort: use-annotation
```
This is intended to only forward calls to /dashboard, which should only allow access to the dashboard and not to other paths of PD.

Reply:
This is actually already documented on https://docs.pingcap.com/tidb-in-kubernetes/stable/access-dashboard but I wasn't aware of this. I added a link in the docs in https://github.com/pingcap/docs-tidb-operator/pull/1212